src/main/kotlin/racingcar/domain/Judge.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar.domain
+
+class Judge {
+    fun judgeWinner(gameResult: List<Car>): List<String> {
+        val winners = mutableListOf<String>()
+
+        var maxDistance = 0
+        for (cars in gameResult) {
+            if (cars.distance == maxDistance) winners.add(cars.name)
+            else if (cars.distance > maxDistance) {
+                winners.clear()
+                winners.add(cars.name)
+                maxDistance = cars.distance
+            }
+        }
+
+        return winners
+    }
```

Comment:
`Judge`의 알고리즘이 복잡하고 indent depth 를 넘어갈 가능성이 있어 보여요😊
Kotlin의 함수형 스타일을 사용하면 더 간결하게 표현이 될 것 같아요.
```suggestion
fun judgeWinner(gameResult: List<Car>): List<String> {
    val maxDistance = gameResult.maxOf { it.distance }
    return gameResult.filter { it.distance == maxDistance }
        .map { it.name }
}
```
이렇게 작성해도 좋을 것 같습니다💪🏻